core/zoom_controls.js

```diff
@@ -30,7 +30,7 @@ Blockly.ZoomControls = function(workspace) {
    */
   this.workspace_ = workspace;
 
-  /**
+  /*
```

Comment:
nit, but I think you need this *

Reply:
Good catch, that was a product of a merge.